### DISCUSSION TITLE
Ability to specify command priority of Enter Key for LexicalTypeaheadMenuPlugin

### DISCUSSION THREAD

**Discussion by johnbonds** [facebook/lexical#3714] [Ideas]
Please see: 
https://stackoverflow.com/questions/73471798/lexical-key-command-conflict-and-priority

Pretty much explains the issue. Basically if I want to use lexical with the type ahead menu plugin to implement mentions AND allow the user to press enter to 'submit' what's in my form, there are conflicts. The stackoverflow question articulates the issue pretty well. A solution other than forking the LexicalTypeaheadMenuPlugin would be ideal.

Thoughts?

**Comment by huw** [comment]
I’ve submitted at PR at #5123
